pyiron_potentialfit/pacemaker/job.py

```diff
@@ -251,13 +256,25 @@ def write_input(self):
         ):  # duck-typing check for TrainingContainer
             logging.info("structure_data is TrainingContainer")
             df = self.structure_data.to_pandas()
-            data_file_name = self._save_structure_dataframe_pckl_gzip(df)
-            self.input["data"] = {"filename": data_file_name}
+            data_file_name = self._save_structure_dataframe_pckl_gzip(df = df)
+            self.input["data"]['filename'] = data_file_name
         elif self.structure_data is None:
             raise ValueError(
                 "`structure_data` is none, but should be pd.DataFrame, TrainingContainer or valid pickle.gzip filename"
             )
 
+        # prepare testing datafile
+        if self._test_job_id_list and self.testing_structure_data is None:
+            test_df = self.create_training_dataframe(self._test_job_id_list)
+            self.testing_structure_data = test_df
+            self.testing_structure_data.index += len(train_df)
```

Comment:
I do not think this line is necessary any longer after the corresponding fix in `python-ace` https://github.com/ICAMS/python-ace/pull/80
```suggestion
```